packages/docs-nextra/scripts/wrap-doenet-example.ts

```diff
@@ -0,0 +1,45 @@
+import { visit } from "unist-util-visit";
+import { Plugin } from "unified";
+
+import { Root as MdastRoot } from "mdast";
+// Importing this automatically imports all the types from `mdast-util-mdx-jsx`.
+import "mdast-util-mdx-jsx";
+
+/**
+ * Wraps codeblocks of type `doenet-example` with a JSX element that renders the code and the output.
+ */
+export const wrapDoenetExample: Plugin<void[], MdastRoot, MdastRoot> =
+    function () {
+        return (tree) => {
+            visit(tree, (node) => {
+                type MdxJsxFlowElement = typeof node & {
+                    type: "mdxJsxFlowElement";
+                };
```

Comment:
This type is used in several files. I think we can put it in a `types.ts` and import it (that file can also import `"mdast-util-mdx-jsx"`

Reply:
I reused the original type name `MdxJsxFlowElement` like the previous code did. Not sure if that is a good idea.